server/api/middleware.go

```diff
@@ -106,23 +121,15 @@ func (s *auditMiddleware) ServeHTTP(w http.ResponseWriter, r *http.Request, next
 		return
 	}
 
-	requestInfo, ok := requestutil.RequestInfoFrom(r.Context())
-	if !ok {
-		log.Error("failed to get request info when auditing")
-		next(w, r)
-		return
-	}
+	// There is no need to check whether requestInfo is available like getting RaftCluster
+	requestInfo, _ := requestutil.RequestInfoFrom(r.Context())
 
 	labels := s.svr.GetServiceAuditBackendLabels(requestInfo.ServiceLabel)
```

Comment:
If there is no requestInfo, we don't need to get it?

Reply:
> If there is no requestInfo, we don't need to get it?

If we open audit or rate-limit switch, there must be a requestInfo.

Reply:
Please update the comment here.